pyproject.toml

```diff
@@ -1,119 +1,106 @@
-[build-system]
-requires = ["setuptools>=61.2", "setuptools-scm"]
-build-backend = "setuptools.build_meta"
-
-[tool.setuptools.packages]
-find = {namespaces = false}
-
-[tool.setuptools.package-dir]
-"" = "."
-
-[tool.setuptools.package-data]
-"*" = [
-    "plugins/*.txt",
-    "plugins/*/*.txt",
-    "plugins/*/*/*.txt",
-    "plugins/*/*/*/*.txt",
-    "plugins/*/*/*/*/*.txt",
-    "plugins/*.yaml",
-    "plugins/*/*.yaml",
-    "plugins/*/*/*.yaml",
-    "plugins/*/*/*/*.yaml",
-    "plugins/*/*/*/*/*.yaml",
-    "schema/*.yaml",
-    "schema/*/*.yaml",
-    "schema/*/*/*.yaml",
-    "schema/*/*/*/*.yaml",
-    "schema/*/*/*/*/*.yaml",
-    "image_utils/ascii_palettes/*",
-]
-
-[tool.setuptools_scm]
-"version_scheme" = "post-release"  # Use current version .postN vs incrementing
-"local_scheme" = "no-local-version"  # Don't include hash, or date, just postN
-"write_to" = "geoips/_version.py"  # Write an actual file
-
-[tool.setuptools.dynamic]
-entry-points = {file = ["entry-points.ini"]}
-
-[project]
-name = "geoips"
-authors = []
+[tool.poetry]
+# required ones
+name = "geoips"         # geoips package available at the root of the project
+version = "0.0.0"   # this has to be fixed; it's dynamic in setuptools
```

Comment:
Does this need to be removed now, with the poetry dynamic versioning below?

Reply:
Actually, poetry uses `"0.0.0"` as a known placeholder so this is correct.

Reply:
The placeholder is needed for the dynamic versioning to work.